lib/wallaroo/core/sink/kafka_sink/kafka_sink_config.pony

```diff
@@ -73,7 +73,8 @@ class val KafkaSinkBuilder
   fun apply(sink_name: String, event_log: EventLog,
     reporter: MetricsReporter iso, env: Env,
     barrier_coordinator: BarrierCoordinator, checkpoint_initiator: CheckpointInitiator,
-    recovering: Bool, worker_name_: WorkerName, auth_: AmbientAuth): Sink
+    recovering: Bool, app_name_: String, worker_name_: WorkerName,
```

Comment:
Why `app_name_`?  Is this because of shadowing?  In that case, the convention is `app_name'`.  Same for `worker_name_` and `auth_`.